### PR TITLE
[WIP] Added Streamgraph example

### PR DESCRIPTION
**Pull request by ayush29feb**
Added the streamgraph example (showInEditor : false)

### CODE REVIEW

**Comment by domoritz** [comment]
Maybe we can replace this with a movie example.

**Comment by kanitw** [comment]
You can say "fixes #888" to suggest that this fixes an issue.  ;)

File: examples/streamgraph.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      "field": "Weight_in_lbs",
+      "type": "quantitative",
+      "aggregate": "sum",
+      "axis": {"labels": false,"grid": false,"tickSize": 0}
+    },
+    "color": {"field": "Cylinders","type": "nominal"}
+  },
+  "config": {
+    "mark": {"interpolate": "cardinal"},
+    "stack": {"offset": "center"}
```
Comment (kanitw) [inline examples/streamgraph.json]: Actually after thinking more about it, the offset should be `”wiggle”`, so it’s similar to the d3 example: http://bl.ocks.org/mbostock/4060954.

**Comment by kanitw** [comment]
@domoritz  I don’t know if this really needs to become the movie example.

**Comment by domoritz** [comment]
I think the move dataset leads to an infinitely more interesting visualization. Seeing the weights of cars by # of cylinders over time is not that exciting.

**Comment by ayush29feb** [comment]
Hey, so I have been trying some examples for the movies data set. However, I am not really able to get a good looking visualization. I tried stuff like `"x": "Release_Date"`, `y: US_Gross` and `color : Major_Genre`
I am guessing this is because the data is not very consistent and contains a lots of `null`

image

**Comment by domoritz** [comment]
Marked as WIP. @ayush29feb change title when you're done.

**Comment by kanitw** [comment]
supersedes by https://github.com/vega/vega-lite/pull/1138

**Pull request closed without merging**